src/client.ts

```diff
@@ -129,12 +134,16 @@ export class Original {
   errorFromResponse(response: AxiosResponse<APIErrorResponse>): ErrorFromResponse<APIErrorResponse> {
     let err: ErrorFromResponse<APIErrorResponse>;
     err = new ErrorFromResponse(`GetOriginal error HTTP code: ${response.status}`);
-    if (response.data && response.data.code) {
-      err = new Error(`GetOriginal error code ${response.data.code}: ${response.data.message}`);
-      err.code = response.data.code;
+    if (response.data && response.data.error && response.data.error.detail && response.data.error.type) {
+      err = new Error(
+        `GetOriginal error code ${response.status}: ${response.data.error.type}: ${JSON.stringify(
```

Comment:
not related but same "consistency" topic

In typescript we do

```javascript

const client = new Original('YOUR_API_KEY', 'API_KEY_SECRET');
```

In python we do

```python

client = OriginalClient(api_key='YOUR_API_KEY', api_secret='API_KEY_SECRET')
```

Would be nice to align both.

Reply:
just raised the release for python, which do we prefer, Original or OriginalClient?

Reply:
Wont make the change Miguel mentioned right now, as it would require a full version bump. Meaning Neopolis/YayPal/Sekai would have to change the code base

Reply:
I would be tempted to prefer the typescript version because I believe that Neopolis using it but I am not sure.
If we change the python sdk, we must notify yaypal though...

What you could do is having both for back compatibility and `OriginalClient` calling `Original` (then update the doc)

WDYT guys?

Reply:
I'll do this in a follow up PR, just to keep changeLog and the linked linear tickets in sync